### DISCUSSION TITLE
Github email notification outage?

### DISCUSSION THREAD

**Discussion by rossberg** [WebAssembly/spec#1456] [General]
It seems as if GH email notifications have not worked properly yesterday (during European day time), or perhaps longer, since I received replies this morning to earlier posts that I did not receive. I also did not receive copies of my own replies yesterday, like I always did (and now do again). This happened for all Wasm repos, AFAICT.

Has anybody else observed this? Has anybody seen email notifications for the various replies I posted yesterday?

**Comment by tlively** [comment]
I have email notifications of your comments https://github.com/WebAssembly/exception-handling/issues/206#issuecomment-1109575189 and https://github.com/WebAssembly/design/issues/1395#issuecomment-1109606495, which might be during the period of time in question. Before that I don't see anything from you since April 14.

**Comment by rossberg** [comment]
Thanks for the check @tlively. Very odd, I did not get those. So let's hope it just affected my account then. (With me on vacation, I didn't send anything before that day.)
